sycl/source/handler.cpp

```diff
@@ -978,7 +978,7 @@ void handler::memset(void *Dest, int Value, size_t Count) {
   MDstPtr = Dest;
   MPattern.push_back(static_cast<char>(Value));
   MLength = Count;
-  setUserFacingNodeType(ext::oneapi::experimental::node_type::memset);
+  setUserFacingNodeType(ext::oneapi::experimental::node_type::memfill);
```

Comment:
I don't think this is correct, [setUserFacingNodeType](https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/handler.hpp#L1845) is for setting the type to match what the user asked for, and differentiate from the implementation detail

Reply:
Makes sense, I've changed it now back to `node_type::memset`